Remotion/Web/Development.WebTesting/HostingStrategies/DockerHostingStrategy.cs

```diff
@@ -52,7 +52,8 @@ public class DockerHostingStrategy : IHostingStrategy
       var resources = testSiteLayoutConfiguration.Resources.Select (resource => resource.Path);
       var mounts = resources
           .Select (path => GetAbsolutePath (path, absoluteWebApplicationPath))
-          .Distinct (StringComparer.OrdinalIgnoreCase);
+          .Distinct (StringComparer.OrdinalIgnoreCase)
+          .ToArray();

```

Comment:
Ah okay. Good that we changed this, otherwise, we would have a) lazily built the input parameter only once needed and b) re-evaluated the building of the input parameter if we used it multiple times.